livre/Lib/Uploaders/DataUploadProcessor.cpp

```diff
@@ -200,47 +197,35 @@ void DataUploadProcessor::runLoop_( )
 
 }
 
-bool DataUploadProcessor::_loadPrioritizedData( const Frustum& frustum,
-                                                const LoadPriority priority )
+void DataUploadProcessor::_loadData()
 {
+    const DashRenderStatus& renderStatus = _dashTree->getRenderStatus();
+
+    const Frustum& frustum = renderStatus.getFrustum();
+    _currentFrameID = renderStatus.getFrameID();
+
+    DFSTraversal traverser;
```

Comment:
_loadPrioritizedData is long gone, long time ago, far far away galaxy there was lots of code for prefetching which is currently useless.